src/main/webapp/js/index.js

```diff
@@ -127,3 +127,79 @@ $(function() {
           cb);
   cb(start, end);
 });
+
+
+/**
+ * Price range slider setup:
+ * https://refreshless.com/nouislider/.
+ */
+$(document).ready(function() {
+  const keypressSlider = document.querySelector('.slider-keypress');
+  const input0 = document.querySelector('.input-with-keypress-0');
+  const input1 = document.querySelector('.input-with-keypress-1');
+  const inputs = [input0, input1];
+
+  noUiSlider.create(
+      keypressSlider,
+      {start: [20, 80], connect: true, step: 1, range: {min: [0], max: [250]}});
+
+  keypressSlider.noUiSlider.on('update', function(values, handle) {
+    inputs[handle].value = values[handle];
+
+    /* begin Listen to keypress on the input */
+    function setSliderHandle(i, value) {
+      const r = [null, null];
+      r[i] = value;
+      keypressSlider.noUiSlider.set(r);
+    }
+
+    // Listen to keydown events on the input field.
+    inputs.forEach(function(input, handle) {
+      input.addEventListener('change', function() {
+        setSliderHandle(handle, this.value);
+      });
+
+      input.addEventListener('keydown', function(e) {
+        const values = keypressSlider.noUiSlider.get();
+        const value = Number(values[handle]);
+        const steps = keypressSlider.noUiSlider.steps();
+        const step = steps[handle];
+        let position;
+
+        // 13 is enter, 38 is key up, 40 is key down.
+        switch (e.which) {
+          case 13:
```

Comment:
These values can be extracted to constants to make the code more self-documenting

https://drive.google.com/file/d/1uzaIjN8oLqdVL6Sp0VE7ZnBoehEfg_rz/view